### ISSUE TITLE
Invalid librdkafka version 901ff, expected at least 903ff

### ISSUE THREAD

**Issue by auserx** [ah-/rdkafka-dotnet#124]
Hello

I'm getting this error as soon as I hit this line from the Cosumer Basic Example:

`using (var consumer = new Consumer<Null, string>(config, null, new StringDeserializer(Encoding.UTF8)))`

**Invalid librdkafka version 901ff, expected at least 903ff**

I'm using version 0.95 

Any idea why?

Thanks

**Comment by treziac** [comment]
Edit: you mix rdkafka-dotnet (not anymore maintained) and [Confluent.Kafka](https://github.com/confluentinc/confluent-kafka-dotnet) where it was moved

What framework do you target on which platform?
On classic net framework, be sure to NOT use LibRdkafka.Internals. Just target confluent.Kafka 0.9.5, and if this does not work, add librdkafka.redist v0.9.5 and report the issue
Also check that you didn't manually copy the librdkafka.dll in your bin folder
Thanks!

**Comment by auserx** [comment]
It seems I had an earlier version installed (0.9.1)

I removed and I got an error (missing librdkafka.redist DLL)

So I installed the latest version of librdkafka.redist and now it seems to go beyond the previous line of code.

My new error is:
**4|2017-04-24 17:26:43.113|rdkafka#consumer-1|BROKER| [thrd:app]: Broker name "http://kafka1.pp-bigdata.[myKafka]:[portnumber]" parse error: unsupported protocol "HTTP"**

**Comment by auserx** [comment]
so I removed the http and now it works : )

**Comment by treziac** [comment]
Yep, it's just tcp ;) And you should only have reference to confluent.kafka 0.9.5
If you have any other problems, please post it on confluent repo so we can track it more easily :)

**Comment by auserx** [comment]
If I remove the reference to  librdkafka.redist v0.9.5 I get the following error:

**Unable to load DLL 'librdkafka': The specified module could not be found. (Exception from HRESULT: 0x8007007E)**

on this line:
`using (var consumer = new Consumer<Null, string>(config, null, new StringDeserializer(Encoding.UTF8)))`
